rotkehlchen/api/v1/encoding.py

```diff
@@ -720,6 +720,15 @@ class ModifiableSettingsSchema(Schema):
     active_modules = fields.List(fields.String(), missing=None)
     frontend_settings = fields.String(missing=None)
     account_for_assets_movements = fields.Bool(missing=None)
+    periodic_client_query_period = fields.Integer(
+        strict=True,
+        validate=webargs.validate.Range(
+            min=5,
```

Comment:
Can you make these limits constants, add them to the file, and then reference them in the error message? If you prefer I can also do it.

Reply:
ok let me check it